acquire/utils.py

```diff
@@ -101,6 +112,7 @@ def create_argument_parser(profiles: dict, modules: dict) -> argparse.ArgumentPa
     parser.add_argument("-l", "--log", type=Path, help="log directory location")
     parser.add_argument("--no-log", action="store_true", help=argparse.SUPPRESS)
     parser.add_argument("-p", "--profile", choices=profiles.keys(), help="collection profile")
+    parser.add_argument("--volatile", choices=volatile.keys(), default="none", help="volatile profile")
```

Comment:
--volatile-profile might be more clear otherwise a user enters --volatile and gets.... no volatile.